code/style.css

```diff
@@ -1,7 +1,387 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #270604;
+}
+
+main {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 100px;
+  margin-bottom: 50px;
+  padding: 30px;
+}
+
+/* Hero */
+
+.header {
+  position: relative;
+}
+
+.hero-video {
+  width: 100vw;
+  height: 50vh;
+  object-fit: cover;
+}
+
+.hero-image {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh;
+  width: 100vw;
+}
+
+img {
+  width: 782px;
+  height: 274px;
```

Comment:
We should avoid specifying both width and height - it can lead to loosing proper ratio of an image. width is completely enough